src/IBM.Watson.CompareComply.v1/IBM.Watson.CompareComply.v1.csproj

```diff
@@ -25,8 +25,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="IBM.Cloud.SDK.Core" Version="0.8.0" />
-        <PackageReference Include="IBM.Watson.Common" Version="3.0.1" />
+        <PackageReference Include="IBM.Cloud.SDK.Core" Version="1.0.0" />
+        <PackageReference Include="IBM.Watson.Common" Version="4.0.0" />

```

Comment:
You could change the `IBM.Watson.Common` versions in these service `.csproj` files too just for consistency, but bumpversion will pass fine since there is a `3.5.0` string in these files for the services themselves.